packages/sdk-ts/src/client/exchange/rest/ExchangeRestExplorerApi.ts

```diff
@@ -355,6 +355,26 @@ export class ExchangeRestExplorerApi extends BaseRestConsumer {
         ExchangeRestExplorerTransformer.CW20BalanceToExplorerCW20Balance,
       )
     } catch (error: any) {
+      if (error.message.includes(404) || error.message.includes(500)) {
```

Comment:
We should have a `fetchCW20BalancesNoThrow` version of the function where the error is handled and use that where we need to silently ignore these errors. Also, are we sure we should return a blank contract address and not `undefined`?

Reply:
updated to add a fetchCW20BalancesNoThrow method. regarding `contractAddress`, I discussed with @ThomasRalee  earlier, and `contractAddress` should always be string - since we wont have cw20 balance with undefined contractAddress